src/Ledger/Notation.lagda

```diff
@@ -0,0 +1,24 @@
+\section{Notation}
+\begin{code}[hide]
+{-# OPTIONS --safe #-}
+
+module Ledger.Notation where
+\end{code}
+
+In this section, we introduce some notations used in this document.
+
+\begin{description}
+\item[Propositions, sets and types] This document loosly treats sets and types as the same thing.
+\item[Sums and products] The sum of \AgdaBound{A} and \AgdaBound{B} (also called their disjoint
+  union, coproduct, etc.) is denoted by \AgdaBound{A} \coproduct \AgdaBound{B}, and their product
+  is denoted by \AgdaBound{A} \agdatimes \AgdaBound{B}.
+\item[Record types] Record types are explained in Appendix \ref{sec:appendix-agda-essentials}.
+\item[Postfix projections] TODO, if we want them in the PDF
```

Comment:
Add a normal sentence for postfix projections (e.g. "Record fields are simply projection functions from a value of the record to said field and can be conveniently written in postfix form; for instance, getting the e-mail `address` of a `user` would be denoted `user .address`"); we can then remove it if we decide no such projections are ever rendered on PDF.